biggraphite/glob_utils.py

```diff
@@ -254,6 +255,40 @@ def maybe_matched_prefilter(metric):
     return list(filter(maybe_matched_prefilter, metric_names))
 
 
+class GlobMetricResult:
```

Comment:
I don't know if this intermediate class is really necessary .. but worst case it can't really hurt.

Reply:
I wanted to avoid returning too complex anonymous types (tuple of lists of key-value pairs). This sounded more readable to me.

Reply:
ok :+1: